Auth/Auth.Application/Services/HttpClientFactoryService.cs

```diff
@@ -5,13 +5,13 @@ namespace Auth.Application.Services
 {
     public class HttpClientFactoryService : IHttpClientFactoryService
     {
-       private readonly IHttpClientFactory _httpClientFactory;
-        private readonly JsonSerializerOptions _options;
+       private readonly IHttpClientFactory httpClientFactory;
+        private readonly JsonSerializerOptions options;
         
         public HttpClientFactoryService(IHttpClientFactory httpClientFactory)
         {
-            _httpClientFactory = httpClientFactory;
-            _options = new JsonSerializerOptions { PropertyNameCaseInsensitive = true };
+            this.httpClientFactory = httpClientFactory;
+            options = new JsonSerializerOptions { PropertyNameCaseInsensitive = true };
```

Comment:
we have an ability to directly configure and inject `httpClient` instead of `httpClientFactory`